sotopia/envs/evaluators.py

```diff
@@ -99,7 +109,10 @@ class EpisodeLLMEvaluator(Evaluator, Generic[T_eval_dim]):
     def __init__(
         self,
         model_name: str,
-        response_format_class: type[EvaluationForTwoAgents[T_eval_dim]],
+        response_format_class: Union[
+            type[EvaluationForTwoAgents[T_eval_dim]],
```

Comment:
Could we merge `EvaluationForTwoAgents` and `EvaluationForMultipleAgents` together? those two classes should basically serve as the same functionality?